printer_mods/Printopal/Magnetic_top_panel/readme.md

```diff
@@ -0,0 +1,45 @@
+# Magnetic top panel
+# Overview
+This top panel can be easily removed for printer maintenance or giving more cooling.
+3D models are designed to match your buildplate size.
+For versions 1.8 and 2.4 in 250, 300 and 350mm.
+
+https://user-images.githubusercontent.com/85416803/120906040-da129680-c61b-11eb-813c-af05dc87f28a.mp4
+
+# BOM
+250:  
+400 x 400 x 3mm acrylic panel sheet  
+4x 344mm of 6 x 11,7mm [1/2” x 1/4”] High density foam
+
+300:  
+450 x 450 x 3mm acrylic panel sheet  
+4x 394mm of 6 x 11,7mm [1/2” x 1/4”] High density foam
+
+350:  
+500 x 500 x 3mm acrylic panel sheet  
+4x 444mm of 6 x 11,7mm [1/2” x 1/4”] High density foam
+
+Core:  
+8x Ø6 x 3mm neodymium magnet
+
+Acrylic sheet with abrasion resistant coating may be considered for V1.8 to prevent micro scratch caused by the reverse bowden tube. This may also appear with full height prints on V2.4.
+I don’t recommend using glass plate because of its weight, this may result in broken top skirt and broken glass!
+
+
+# Instructions
+1. Insert magnets into panel corner and frame corner respecting their polarity.
+2. Insert frame corner on top of the 4 vertical 2020 extrusions.
+3. Put panel corner on frame corner, sides should be flush.
+4. Glue the acrylic panel sheet on panel corners, be sure to keep panel corner and frame corner sides flush while gluing the acrylic panel.
+5. Remove the panel from the frame, adjust foam length and glue it under the acrylic panel sheet (same side as panel corner).
+6. Glue top corner skirt on the acrylic panel sheet.
+7. Adjust top skirt length and glue on the acrylic panel sheet. You can easily adjust the length in your slicer.
+#
+![Magnetic_top_panel](https://user-images.githubusercontent.com/85416803/120905966-53f65000-c61b-11eb-8367-2481f096431a.jpg)
```

Comment:
Can you upload all the images from this README into a directory under your mod (maybe `printer_mods/Printopal/Magnetic_top_panel/images/`) and change the references in this README to use those uploaded images?  For example, if this image was uploaded as `images/panel.jpg`, then your image would be

    ![Magnetic_top_panel](images/panel.jpg)

Reply:
Thank for your feedback! I created an Images folder and modified their path in README as you suggested.
I couldn't get the same thing working for the video, any advices?

Reply:
I could not find a way to do this with a video.  So you can either leave it as you have it now, or upload it to the directory and just link to it - but then it won't play embedded, it'll just make the user download it.

Can you also comment on my question about glue/VHB tape?

Reply:
Thanks, I will let the video like this so.
I missed that question sorry, it's all done now.